build/config/compiler/BUILD.gn

```diff
@@ -822,9 +843,10 @@ config("optimize") {
     cflags = [ "-Os" ] + common_optimize_on_cflags  # Favor size over speed.
   } else if (is_android) {
     cflags = [ "-Oz" ] + common_optimize_on_cflags  # Favor size over speed.
-    if (enable_lto) {
-      lto_flags += [ "-flto" ]
-    }
+    # TODO(goderbauer): re-enable when  https://github.com/flutter/flutter/issues/21686 is fixed
```

Comment:
This looks like a bad merge? I think LTO should stay on.

Reply:
Good catch! I think I was too far behind in the tree. Lemme go back and fix that.